src/solver/variable/economy/bindingConstraints/bindingConstraintsMarginalCost.h

```diff
@@ -392,7 +393,7 @@ class BindingConstMarginCost
     //! Intermediate values for each year
     typename VCardType::IntermediateValuesType pValuesForTheCurrentYear = nullptr;
     unsigned int pNbYearsParallel = 0;
-    Data::BindingConstraintRTI* associatedBC_ = nullptr;
+    std::shared_ptr<Data::BindingConstraint> associatedBC_ = nullptr;
```

Comment:
About `associatedBC_ = nullptr;` : 
Do we need to initialize a shared pointer to **nullptr** ? 
It's its default value anyway, isn't it ?

Reply:
Yes

Reply:
See #1382